### ISSUE TITLE
Upgrade Busybox to >=1.35.0-r17 for fixing the CVE-2022-30065

### ISSUE THREAD

**Issue by vienleidl** [Cisco-Talos/clamav#660]
Alpine Linux has released a security update for busybox to fix the vulnerabilities. The affected package versions are prior to 1.35.0-r17 [(CVE-2022-30065)](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-30065)  Would it be possible to update the busybox-1.35.0-r15 to >=1.35.0-r17? 

The GHSA link is [here](https://github.com/advisories/GHSA-gq73-rh3m-3php) for more information.

**Comment by micahsnyder** [comment]
We're using `alpine:latest` for the base for our image. At this moment, that comes with 1.35.0-r15.  On the image, if I `apk update && apk add busybox --upgrade`, I still only get 1.35.0-r16.  

It appears we have to wait for Alpine to update their packages and images.

**Comment by nishant-yt** [comment]
Any idea by when can we expect the fix ?

**Comment by vienleidl** [comment]
> We're using `alpine:latest` for the base for our image. At this moment, that comes with 1.35.0-r15. On the image, if I `apk update && apk add busybox --upgrade`, I still only get 1.35.0-r16.
> 
> It appears we have to wait for Alpine to update their packages and images.

The [busybox CVE-2022-3065](https://git.alpinelinux.org/aports/commit/?h=v3.16.1&id=e58f0e341603e9facc1e675722367174b45e0d0c) was fixed on Alpine Linux with Releases: [3.16.1](https://www.alpinelinux.org/posts/Alpine-3.16.1-released.html), [3.13.11, 3.14.7 AND 3.15.5](https://www.alpinelinux.org/posts/Alpine-3.13.11-3.14.7-3.15.5-released.html).

**Comment by vienleidl** [comment]
It's fixed in the new release of [Alpine Linux 3.16.2](https://alpinelinux.org/posts/Alpine-3.13.12-3.14.8-3.15.6-3.16.2-released.html). Should we upgrade the Clamav image to use 3.16.2 as well?

```
docker run --rm alpine:3.16 cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.16.2
PRETTY_NAME="Alpine Linux v3.16"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
```

```
docker run --rm alpine:3.16 apk busybox info
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/main: No such file or directory
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/community: No such file or directory
busybox-1.35.0-r17 description:
Size optimized toolbox of many common UNIX utilities

busybox-1.35.0-r17 webpage:
https://busybox.net/

busybox-1.35.0-r17 installed size:
940 KiB
```

```
docker run --rm clamav/clamav apk busybox info
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/main: No such file or directory
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/community: No such file or directory
busybox-1.35.0-r15 description:
Size optimized toolbox of many common UNIX utilities

busybox-1.35.0-r15 webpage:
https://busybox.net/

busybox-1.35.0-r15 installed size:
936 KiB
```

**Comment by micahsnyder** [comment]
We just published a images for 0.104.4 and 0.105.1  (0.104.4-2, 0.104.4-2_base, 0.105.1-2, and 0.105.1-2_base).  The 0.104, 0.105, and latest tags have all been updated to point to the new images.
